tsz-compress/src/v2/halfvec.rs

```diff
@@ -185,6 +185,25 @@ impl HalfVec {
     }
 }
 
+/// Compares two HalfVec for equality.
+impl PartialEq for HalfVec {
```

Comment:
This is not a valid implementation of PartialEq. We would not want to provide this as a library feature due to the expense of a correct check.

If this is for use in tests, then we can provide a function in the tests module.

Reply:
Changes in commit [c19be9f](https://github.com/qsib-cbie/tsz/pull/36/commits/c19be9f42ebca7fe539b84512e03f6fb671f51db).